src/server/api/cycleData/table/estimateValues.ts

```diff
@@ -68,8 +68,8 @@ export const estimateValues = async (req: CycleDataRequest<never, EstimateBody>,
     const estimation = generateSpecToEstimation({ generateSpec, table })
     const nodes = EstimationEngine.estimateValues(
       years,
-      originalDataPointValues,
-      generateSpec,
+      originalDataPointValues[assessment.props.name][cycle.name],
+      generateSpec as GenerateSpec,
```

Comment:
cast in unnecessary ? command util method to extract       originalDataPointValues[assessment.props.name][cycle.name] ?